core/mgmt/v1beta/metric.proto

```diff
@@ -91,44 +89,42 @@ message GetPipelineTriggerCountResponse {
   repeated PipelineTriggerCount pipeline_trigger_counts = 1;
 }
 
-/*
-   // ListPipelineTriggerChartRecordsRequest represents a request to list pipeline
-   // trigger chart records for a given requester, grouped by time buckets.
-   message ListPipelineTriggerChartRecordsRequest {
-     // 1 is reserved for the aggregation window in nanoseconds. This is
-     // deprecated in favour of an aggregation window string that represents a
-     // duration.
-     reserved 1;
-     // 2 is reserved for the filter. For now, this endpoint won't allow filtering
-     // but in the future we might implement a filter to show the trigger count of
-     // only certain pipelines and to group by the pipeline ID.
-     reserved 2;
+// ListPipelineTriggerChartRecordsRequest represents a request to list pipeline
+// trigger chart records for a given requester, grouped by time buckets.
+message ListPipelineTriggerChartRecordsRequest {
+  // 1 is reserved for the aggregation window in nanoseconds. This is
+  // deprecated in favour of an aggregation window string that represents a
+  // duration.
+  reserved 1;
+  // 2 is reserved for the filter. For now, this endpoint won't allow filtering
+  // but in the future we might implement a filter to show the trigger count of
+  // only certain pipelines and to group by the pipeline ID.
+  reserved 2;
 
-     // The ID of the namespace that requested the pipeline triggers.
-     string namespace_id = 3 [(google.api.field_behavior) = REQUIRED];
-     // Aggregation window. The value is a positive duration string, i.e. a
-     // sequence of decimal numbers, each with optional fraction and a unit
-     // suffix, such as "300ms", "1.5h" or "2h45m".
-     // The minimum (and default) window is 1h.
-     optional string aggregation_window = 4;
-     // Beginning of the time range from which the records will be fetched.
-     // The default value is the beginning of the current day, in UTC.
-     optional google.protobuf.Timestamp start = 5;
-     // End of the time range from which the records will be fetched.
-     // The default value is the current timestamp.
-     optional google.protobuf.Timestamp stop = 6;
-   }
+  // The ID of the namespace that requested the pipeline triggers.
+  string namespace_id = 3 [(google.api.field_behavior) = REQUIRED];
```

Comment:
Perhaps this should be `requester_id` in order not to mix `namespace` (the owner of the pipeline triggers) with `requester`.

Also, is `trigger` the right term anymore or should we use `run` when documenting this fields?

cc @donch1989 @joremysh

Reply:
> Perhaps this should be requester_id in order not to mix namespace (the owner of the pipeline triggers) with requester.

It was agreed to use `requester_id` for this parameter 🤝